apps/omg_child_chain/lib/omg_child_chain/fee_server.ex

```diff
@@ -49,7 +49,7 @@ defmodule OMG.ChildChain.FeeServer do
       |> Map.put(:expire_fee_timer, nil)
       |> update_fee_specs()
```

Comment:
also, don't call update_fee_specs from the `init`

remember that `init` needs to be *pure* if you want your process to be restartable

Reply:
So that is another argument that `update_fee_specs` won't crash but return an error

Reply:
ahaaaa! ok, so the question is: in what state does Fee Server leave our entire system if on boot `update_fee_specs()` returns:
```
_ = Logger.error("Unable to update fees from file. Reason: #{inspect(error)}")
```

Reply:
I see your point - I'm making assumption that I'll successfully get the initial state more likely than the later ones. 
I think I can reschedule fee update just after init and make sure we handle unsuccessful update gracefully.

Reply:
I think I'll do the change in the subsequent PR to not block this one, and also make a reversion easier of possible regression.

Reply:
What do you mean by that?

Reply:
Doh, I put my comment in a wrong thread 🤦‍♂

Reply:
> remember that init needs to be pure if you want your process to be restartable
Regarding above, it's out-of-scope this pr, but I agree it improves system reliability. 
We can track it in the task